pytest_cases/main_fixtures.py

```diff
@@ -113,10 +121,12 @@ def _create_fixture(param_idx):
             @with_signature("(%s)" % root_fixture_name)
             def _param_fixture(**kwargs):
                 params = kwargs.pop(root_fixture_name)
-                return params[param_idx] if len(argnames_lst) > 1 else params
+                return params[param_idx]
```

Comment:
Also returning the value instead of a tuple with a single value. This is consistent with the way `pytest.mark.parametrize` works.

Reply:
Good catch. It seems that you used `param_fixtures` (plural) with a single argname, that's why you fell into that case while it was never hit in the tests.